backbone/README.md

```diff
@@ -2,23 +2,370 @@
+	}
+});
+
+// bad (uses magic numbers/strings)
+var $ = require('jquery'),
+	Marionette = require('marionette');
+
+return Marionette.ItemView.extend({
+	initialize: function(options) {
+		$('someDynamicSelector').addClass('is-hidden');
+		window.setTimeout(this.someCallback, 10);
+	}
+});
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## Styling
+
+To simplify searches when trying to find templates, put CSS classes in handlebars templates instead of coupling it with the view logic:
+
+```js
+// good
+
+// some_view.handlebars
+
+<div class="g-cell g-cell-12-12"></div>
+
+// some_view.js
+
+var Marionette = require('marionette'),
+	template = require('hb!./some_view.handlebars');
+
+return Marionette.ItemView({
+	template: template
+});
+
+
+// bad (CSS classes aren't separated out)
+var Marionette = require('marionette');
+
+return Marionette.ItemView({
+	classname: 'g-cell g-cell-12-12'
+});
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+## Context
+
+### Binding
+
+In order to use native JavaScript whenever possible, use [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind) instead of [`_.bind`](http://underscorejs.org/#bind) and [`_.bindAll`](http://underscorejs.org/#bindAll) to bind callback handlers:
+
+```js
+// good
+return Marionette.ItemView.extend({
+	initialize: function(options) {
+		this.listenTo(channel.vent, 'someSignal', this.someMethod.bind(this));
+		this.listenTo(channel.vent, 'anotherSingle', this.anotherMethod.bind(this));
+	},
+
+	someMethod: function(options) {
+		/* do something */
+	},
+	anotherMethod: function(options) {
+		/* do something */
+	}
+});
+
+// bad (uses _.bindAll)
+return Marionette.ItemView.extend({
+	initialize: function(options) {
+		_.bindAll(this, 'someMethod', 'anotherMethod');
+
+		this.listenTo(channel.vent, 'someSignal', this.someMethod);
+		this.listenTo(channel.vent, 'anotherSingle', this.anotherMethod);
+	},
+
+	someMethod: function(options) {
+		/* do something */
+	},
+	anotherMethod: function(options) {
+		/* do something */
+	}
+});
+
+// bad (uses _.bind)
+return Marionette.ItemView.extend({
+	initialize: function(options) {
+		this.listenTo(channel.vent, 'someSignal', _.bind(this.someMethod));
+		this.listenTo(channel.vent, 'anotherSingle', _.bind(this.anotherMethod));
+	},
+
+	someMethod: function(options) {
+		/* do something */
+	},
+	anotherMethod: function(options) {
+		/* do something */
+	}
+});
+```
+
+**[⬆ back to top](#table-of-contents)**
+
+### Data
+
+Storing derived/calculated data on the `this` context of a view can be fragile and error prone because nothing prevents that data from being modified. Furthermore, it makes quality code review (aka static analysis) more challenging as the reviewer needs to first investigate where the instance property originates.
+
+Whenever possible, calculate the data on demand either in the model or in the view:
+
+```js
```

Comment:
I think the code examples here need to be updated.

None of the examples show the computed data being used. The "ok" & "bad" examples use `someTransformation` which isn't shown. Maybe instead use a comment that says "transform options in some way"?